Training25/Program.cs

```diff
@@ -3,11 +3,75 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T2 branch.
+// Program to convert input number to words and roman numerals.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      do {
+         Clear ();
+         Write ("Enter input: ");
+         if (int.TryParse (ReadLine (), out int output) && output > 0) {
+            WriteLine ("Choose option:\n\tA) Convert number to words\n\tB) Convert number to roman numerals");
+            switch (ReadKey (true).Key) {
+               case ConsoleKey.A: WriteLine (ConvertToWords (output)); break;
+               case ConsoleKey.B: WriteLine (ConvertToRoman (output)); break;
+            }
+         } else
+            Console.WriteLine ("Please enter a positive integer!");
+         WriteLine ("\nPress 'Y' to continue");
+      } while (ReadKey (true).Key == ConsoleKey.Y);
+   }
+
+   /// <summary>Converts input number to words</summary>
+   public static string ConvertToWords (int num) {
+      string[] ones = { "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine" },
+         teen = { "Eleven", "Twelve", "Thirteen", "Fourteen", "Fifteen", "Sixteen",
+         "Seventeen", "Eighteen", "Nineteen" },
+         tens = { "Ten", "Twenty", "Thirty", "Fourty", "Fifty", "Sixty", "Seventy", "Eighty",
+         "Ninety" },
+         baseVals = { "", "Thousand", "Mil1ion", "Billion" };
+      string output = "";
+      for (int baseVal = 1000000000, indexPos = 3; baseVal >= 1 && indexPos >= 0; baseVal /= 1000, indexPos--) {
+         int quotient = num / baseVal;
+         // Group number into hundreds
+         while (quotient > 0) {
+            switch (quotient) {
+               case int n when (n > 100):

```

Comment:
```suggestion
            switch (quotient) {
               case > 100:
```
C# 9 allows us to do this. https://stackoverflow.com/questions/20147879/switch-case-can-i-use-a-range-instead-of-a-one-number